setup.cfg

```diff
@@ -38,3 +38,6 @@ setup_requires = pytest-runner==4.4
 
 [aliases]
 test = pytest
+
+[metadata]
```

Comment:
Running `python setup.py sdist` failed with the following error:

```bash
$ python setup.py sdist
Traceback (most recent call last):
  File "setup.py", line 62, in <module>
    "Tracker": "https://github.com/cylc/cylc-xtriggers/issues"
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
    _install_setup_requires(attrs)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 137, in _install_setup_requires
    dist.parse_config_files(ignore_option_errors=True)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/dist.py", line 697, in parse_config_files
    self._parse_config_files(filenames=filenames)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/dist.py", line 594, in _parse_config_files
    (parser.read_file if six.PY3 else parser.readfp)(reader)
  File "/home/kinow/anaconda3/lib/python3.7/configparser.py", line 717, in read_file
    self._read(f, source)
  File "/home/kinow/anaconda3/lib/python3.7/configparser.py", line 1065, in _read
    lineno)
configparser.DuplicateSectionError: While reading from 'setup.cfg' [line 42]: section 'metadata' already exists
```

Looks like the `[metadata]` is already declared above this line. Could you please move the `license_files` there @sgaist ? Then I'll do another quick test here :+1: 

Thanks!

Reply:
Late night coding...

I didn't pay enough attention to the file content.

Reply:
No worries, I code better in the morning, but also write some code in the late night... that I end up fixing in the morning :grin: let's merge it! Thanks!